python/core-sdk/alation_ai_agent_sdk/tools.py

```diff
@@ -565,6 +570,203 @@ def run(
+                    "display_names": "Use 'name_singular' for field labels and column headers in user interfaces. Use 'name_plural' when displaying fields that have multiple values selected (when 'allow_multiple' is true). If 'name_plural' is empty, fall back to 'name_singular' or add 's' for pluralization.",
+                    "field_types": "TEXT = single line text, RICH_TEXT = formatted text with HTML, PICKER = single selection dropdown, MULTI_PICKER = multiple selection checkboxes, OBJECT_SET = references to users/groups/objects, DATE = date picker. Use 'field_type' to determine appropriate input validation and UI controls.",
+                    "csv_headers": "For data dictionary CSV files, use a combination of the field's ID and name for the column headers. The required format is id|field_name (e.g., 4|description or 10020|Business Owner). This is required for Alation to recognize which field to update."
+                }
+            }
+        except AlationAPIError as e:
+            return {"error": e.to_dict()}
+
+    def _filter_field_properties(self, raw_fields: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+        """
+        Filter each field to include only the 8 required properties.
+
+        Args:
+            raw_fields: Full field objects from the API
+
+        Returns:
+            List of field objects with only the 8 selected properties
+        """
+        filtered_fields = []
+
+        # The 8 properties we want to keep
+        selected_properties = [
+            'id',
+            'name_singular',
+            'field_type',
+            'allowed_otypes',
+            'options',
+            'tooltip_text',
+            'allow_multiple',
+            'name_plural'
+        ]
+
+        for field in raw_fields:
+            filtered_field = {}
+            for prop in selected_properties:
+                filtered_field[prop] = field.get(prop)
+            filtered_fields.append(filtered_field)
+
+        return filtered_fields
+
+
+class GetDataDictionaryInstructionsTool:
+    """
+    Generates comprehensive instructions for creating Alation Data Dictionary CSV files.
+
+    This tool provides LLMs with complete formatting rules, validation schemas, and examples
+    for transforming object metadata into properly formatted data dictionary CSVs.
+    """
+
+    def __init__(self, api: AlationAPI):
+        self.api = api
+        self.name = self._get_name()
+        self.description = self._get_description()
+
+    @staticmethod
+    def _get_name() -> str:
+        return "get_data_dictionary_instructions"
+
+    @staticmethod
+    def _get_description() -> str:
+        return """
+        Generates comprehensive instructions for creating Alation Data Dictionary CSV files.
```

Comment:
Wondering if we should explain what Data Dictionary CSV files is for, something like `Generates comprehensive instructions for creating Alation Data Dictionary CSV files to update logical metadata of objects`.

In the case customers know that they want to use data dictionary, the current description is fine, but if someone wants to bulk update descriptions or custom fields and not know what tools exist, will this be easily discovered? Maybe the `when to use` section is good enough, but just wanted to see if we should mention it in the one line summary too.